pkg/cluster/pod.go

```diff
@@ -45,6 +47,93 @@ func (c *Cluster) getRolePods(role PostgresRole) ([]v1.Pod, error) {
 	return pods.Items, nil
 }
 
+// enableRollingUpdateFlagForPod sets the indicator for the rolling update requirement
+// in the Pod annotation.
+func (c *Cluster) enableRollingUpdateFlagForPod(pod v1.Pod, msg string) error {
```

Comment:
setRollingUpdateFlag(pod, msg, value) {}

can potentially include value so it could be set to false despite not being used.

Reply:
`markForRollingUpgrade`